src/applications/app.helpers.js

```diff
@@ -13,6 +13,20 @@ export const UNMOUNTING = 'UNMOUNTING';
 export const UNLOADING = 'UNLOADING';
 export const SKIP_BECAUSE_BROKEN = 'SKIP_BECAUSE_BROKEN';
 
+export const AppStatus = {
```

Comment:
Hmm this duplicates code and maybe leads to the question of whether we should move fully to either one way or the other when it comes to app statuses. The duplicate definitions might be confusing to people, and are also unnecessary bytes to add to the bundle (not that many bytes, but still).

I'm hesitant about making a change that could eventually lead to a breaking change (if we remove the other statuses) if the only reason is for autocomplete. 

@frehner what do you think?

Reply:
Having thought about this further, let's remove this. Matching the single-spa api instead of changing the API is what we should do with this typescript definitions PR.